test/sandbox/richard-hendriks.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
hmm, i think i create this file by running `npm run grunt`

should i remove it? @hacksalot